alpine/docker-entrypoint.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/sh
 set -e
 
-# Disabling nginx daemon mode
-export KONG_NGINX_DAEMON="off"
-
-# Setting default prefix (override any existing variable)
-export KONG_PREFIX="/usr/local/kong"
-
-# Prepare Kong prefix
-if [ "$1" = "/usr/local/openresty/nginx/sbin/nginx" ]; then
-	kong prepare -p "/usr/local/kong"
+if [[ "$1" == "kong" && "$2" =~ "start" ]]; then
```

Comment:
shouldn't this be `"$2" == "start"` ?

No shell guru, but doesn't this enable daemon mode on every-but-start command, instead of only start?

Reply:
The intent is to run this branch on `start` and `restart`. See #167 for the fix in which the intent is clearer, which allows us to stick to bourne shell.